code/src/Components/Artist/Artist.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const Artist = props => {
+  return (
+    <a href={props.url} target="_blank" rel="noopener noreferrer">
+      {props.name}
+    </a>

```

Comment:
one little thing here is that when there is more than one artist they are shown flush together, like `Teyana TaylorKehlani` instead of `Teyana Taylor Kehlani`